RHEL/6/input/system/software/integrity.xml

```diff
@@ -176,9 +176,9 @@ have file hashes different from what is expected by the RPM database.
 The hashes of important files like system executables should match the
 information given by the RPM database. Executables with erroneous hashes could
 be a sign of nefarious activity on the system.</rationale>
-<ident cce="27223-7"  stig="RHEL-06-000519" />
+<ident cce="27223-7" stig="RHEL-06-000519" />
 <oval id="rpm_verify_hashes" />
-<ref nist="CM-6(d),SI-7" disa="1496" />
+<ref nist="CM-6(d),SI-7" disa="366" />
```

Comment:
there's something funky happening in the script. This rule maps to CCI-001496, which is RHEL-06-00281 in DISA's content (not RHEL-06-000519).